DOC Error in the notation of the regularization penalties for multinomial logistic regression in the docs (#26517)
scikit-learn/scikit-learn: doc/modules/linear_model.rst

Where :math:`[P]` represents the Iverson bracket which evaluates to :math:`0`
if :math:`P` is false, otherwise it evaluates to :math:`1`. We currently provide four choices

> for the regularization term :math:`r(W)` via the `penalty` argument  (with :math:`n` being the number of features):

Reviewer: Small nit regarding phrasing:

```suggestion
for the regularization term :math:`r(W)` via the `penalty` argument, where :math:`n`
is the number of features.
```
